### DISCUSSION TITLE
MAC Address Converter

### DISCUSSION THREAD

**Discussion by joerivang** [CorentinTh/it-tools#414] [Ideas]
As a network engineer, i'm always converting mac addresses to different notations, can you integrate a tool such like this?

example: https://www.vultr.com/resources/mac-converter/

**Comment by sharevb** [comment]
Hi @joerivang, implemented in #867

**Comment by sharevb** [comment]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
